### PR TITLE
mtx lora segmentation

### PR DESCRIPTION
**Pull request by Pnom**

### CODE REVIEW

File: src/analog/splitBytesToDataSegments.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    if ( segmentsNumber > MAX_SEGMENTS_IN_SESSION ) {
+        throw new Error(`Message is too big for one segmentation session. Length: ${bytes.length}, maxSegmentSize: ${maxSegmentSize}`);
+    }
+
+    while ( position < bytes.length ) {
+        const segment = new DataSegmentBase({
+            segmentationSessionId,
+            segmentIndex,
+            segmentsNumber,
+            last: false,
```
Comment (DarkPark) [inline src/analog/splitBytesToDataSegments.ts:28]:
```suggestion
            isLast: false,
```

File: src/mtx/CommandBinaryBuffer.ts [hunk 2]
```diff
@@ -15,6 +16,12 @@ export interface IFrameHeader {
     destination: number
 }
 
+export const defaultFrameHeader = {
```
Comment (DarkPark) [inline src/mtx/CommandBinaryBuffer.ts:19]:
```suggestion
export const defaultFrameHeader: IFrameHeader = {
```
?

File: src/mtxLora/UnknownCommand.ts [hunk 3]
```diff
@@ -1,49 +1,3 @@
...
-
-        return JSON.stringify({
-            id,
-            data: getStringFromBytes(data, bytesConversionFormat, bytesConversionFormatOptions)
-        });
-    }
-}
+import UnknownCommand from '../mtx/UnknownCommand.js';
 
 export default UnknownCommand;
```
Comment (DarkPark) [inline src/mtxLora/UnknownCommand.ts:3]: export {default as App} from "./App.js"

?

File: src/analog/splitBytesToDataSegments.ts (hunk 1 above)
Comment (Pnom) [reply src/analog/splitBytesToDataSegments.ts]: fixed

File: src/mtx/CommandBinaryBuffer.ts (hunk 2 above)
Comment (Pnom) [reply src/mtx/CommandBinaryBuffer.ts]: fixed

**Review by DarkPark — APPROVED** [review]

**Pull request merged**